### ISSUE TITLE
[BUG] fix progress bar / remove it

### ISSUE THREAD

**Issue by Nadav-Barak** [deepchecks/deepchecks#2242]
**Describe the bug**
![Screen Shot 2023-01-02 at 16 48 54](https://user-images.githubusercontent.com/67195469/210247237-b8f1e1a3-2b1c-4e1e-b52b-615b3cae37e1.png)

To reproduce: run the following 
`from deepchecks.vision.datasets.detection import mask
from deepchecks.vision.suites import data_integrity, train_test_validation, model_evaluation, full_suite
from deepchecks.vision import VisionData

train_data = mask.load_dataset(object_type="VisionData", shuffle=False)
test_data = mask.load_dataset(object_type="VisionData", shuffle=False)

N_SAMPLES = 1000
suite = model_evaluation(n_samples = N_SAMPLES)
suite.run(train_dataset=train_data,test_dataset=test_data,max_samples=N_SAMPLES)
`
